pull_request_template.md

```diff
@@ -0,0 +1,37 @@
+# Description
+
+<!--- Please include a summary of the changes and the related issue. -->
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
+
+## Type of change
+
+**Select Type(s) of change:**
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Documentation update
+
+# How Has This Been Tested?
+
+<!--- Please describe the tests that you ran to verify your changes. -->
+<!--- Provide instructions that can be reproduced. -->
+<!--- Please also list any relevant details for your test configuration. -->
+
+
+**Test Configuration**:
```

Comment:
I am not sure this is completely relevant since we have tests in place to test them in all supported environments.

Reply:
Gotcha. Was thinking about if someone has an issue, then we know what their setup is, but can always ask if it gets to that point